www/app/ScenesController.js

```diff
@@ -566,23 +566,22 @@ define(['app'], function (app) {
 			$("#scenecontent #LevelDiv").hide();
 			if (isLED(SubType)) {
 				$("#ScenesLedColor").show();
-			} else {
-				if (bShowLevel == true) {
-					var levelDiv$ = $("#scenecontent #LevelDiv");
-					levelDiv$.find("option").show().end().show();
+			}
+			if (bShowLevel == true && !isLED(SubType)) { // TODO: Show level combo box also for LED
```

Comment:
@stas-demydiuk What would be a good way to connect level combo-box with rgbww picker here? In timer editor this was already done by you..

Reply:
As there is almost jQuery stuff, I see the only way just to add listeners for each change and manually do the related updates